docs/UsersGuide.asciidoc

```diff
@@ -1017,6 +1019,9 @@ latest = 30
 openSUSE.+x86_64 = 10
 --------------------------------------------------------------------------------
 
+Note that modifications to the file will count against the limit, so if an
+asset was updated within the timespan it will not be removed.
```

Comment:
really? So see my question above please. This could lead to space depletion if "recurringly updated assets" accumulate

Reply:
Yes. That's what #2530 ([#59888](https://progress.opensuse.org/issues/59888) was about. Assets which are untracked but updated need to be kept; we used to keep those indefinitely before.

Reply:
ok then, thx